python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1468,6 +1469,7 @@ def run(self, visitSummaries):
             visitEntry = {}
             visitEntry["visitId"] = visitRow['visit']
             visitEntry["filterName"] = visitRow['physical_filter']
+            visitEntry["band"] = visitRow['band']
```

Comment:
Mix of " and '

Reply:
Keeping it consistent with what's there.